test/integration/test_kubernetes_staging.py

```diff
@@ -0,0 +1,117 @@
+"""Integration tests for Kubernetes pod staging.
+
+This case is way more brittle than the other integration tests distributed with Galaxy.
+This is because it requires Kubernetes, RabbitMQ, and the test itself needs to know what
+the test host IP address will be relative to a container inside Kubernetes in order to
+communicate job status updates back.
+
+For this reason, this test will only work out of the box currently with Docker for Mac,
```

Comment:
Surely not needed for this PR, but https://kind.sigs.k8s.io/ looks interesting for testing purposes.